test/v1/temporal-types.test.js

```diff
@@ -658,6 +659,147 @@ describe('temporal-types', () => {
     expect(() => new neo4j.types.DateTime(1, 2, 3, 4, 5, 6, 7, 8, 'UK')).toThrow();
   });
 
+  it('should convert standard Date to neo4j LocalTime', () => {
```

Comment:
Maybe add some integration tests that incorporate standard date -> temporal type conversions?